VisualPinball.Unity/VisualPinball.Unity/VPT/Ramp/RampBehavior.cs

```diff
@@ -18,5 +19,28 @@ protected override Engine.VPT.Ramp.Ramp GetItem()
 		{
 			return new Engine.VPT.Ramp.Ramp(data);
 		}
+
+		public override ItemDataTransformType EditorPositionType => ItemDataTransformType.TwoD;
+		public override Vector3 GetEditorPosition()
+		{
+			if (data == null || data.DragPoints.Length == 0) {
+				return Vector3.zero;
+			}
+			return data.DragPoints[0].Vertex.ToUnityVector3();
+		}
+		public override void SetEditorPosition(Vector3 pos)
+		{
+			if (data == null || data.DragPoints.Length == 0) {
+				return;
+			}
+
+			var diff = pos.ToVertex3D().Sub(data.DragPoints[0].Vertex);
+			diff.Z = 0f;
+			data.DragPoints[0].Vertex = pos.ToVertex3D();
+			for (int i = 1; i < data.DragPoints.Length; i++) {
+				var pt = data.DragPoints[i];
+				pt.Vertex = pt.Vertex.Add(diff);
+			}

```

Comment:
This is very cool. AFAIK in VP you have to select all the drag points and only then you can move the entire object.